test/ts/unit/collateralized/scenarios/successful_seizure.ts

```diff
@@ -0,0 +1,148 @@
+import { SeizeCollateralScenario, TestContracts } from "../runners/";
+
+// external
+import { BigNumber } from "bignumber.js";
+import * as moment from "moment";
+
+// utils
+import * as Units from "../../../test_utils/units";
+
+const defaultArgs = {
```

Comment:
Really love this `defaultArgs` design pattern

Reply:
💯